static/src/javascripts-legacy/projects/common/modules/email/run-checks.js

```diff
@@ -90,7 +90,7 @@ define([
 
     function obWidgetIsShown() {
         var $outbrain = $('.js-outbrain-container');
-        return $outbrain && $outbrain.length > 0;
+        return $outbrain && $outbrain.children && $outbrain.children.length > 0;
```

Comment:
Does `$outbrain.children` mean anything, if $outbrain is an array?

`$outbrain.children === undefined` ???🤔 

<img width="521" alt="picture 498" src="https://cloud.githubusercontent.com/assets/8607683/22258737/b01f54c0-e25a-11e6-866c-7e1ac7cd95dd.png">